sql/catalyst/src/main/java/org/apache/spark/sql/connector/util/V2ExpressionSQLBuilder.java

```diff
@@ -102,6 +102,12 @@ public String build(Expression expr) {
         case "FLOOR":
         case "CEIL":
         case "WIDTH_BUCKET":
+        case "SUBSTRING":
+        case "UPPER":
+        case "LOWER":
+        case "TRANSLATE":
+        case "TRIM":
+        case "OVERLAY":
```

Comment:
Could you investigate the syntax of `OVERLAY` ? we need treat it in special way.

Reply:
Yes. I will treat it in special way.